### ISSUE TITLE
Loss of focus is observed while navigating from "WPF" to "Create Workbook" button using Tab.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#328]
### Pre-Requisite:
Follow the instructions for installing the app from 
https://docs.microsoft.com/en-us/xamarin/tools/workbooks/install?tabs=windows 

### Steps to Reproduce
1. Launch Xamarin Workbooks application.
2. Select the working platform to "WPF".
3. Navigate from "WPF"  to "Create Workbook" button using tab in create new workbook window.

### Expected Behavior
No Loss of Focus should be observed 
### Actual Behavior
Loss of focus is observed while navigating from "WPF" to "Create Workbook" button using tab in create new workbook window.
### Note: (Inspect Observation)
Forward Navigation: Focus remains 2 times on "WPF" option by tabbing.
Backward Navigation: Focus remains 2 times on "Create workbook" button on "Shift + Tab"
### User Impact
People with visual impairments can become disoriented when keyboard navigation takes focus somewhere unexpected or when they cannot easily find the content surrounding an interactive element.
### Environment
Windows : Version 1803 (OS Build : 17134.48)
Xamarin Version : 1.4(master/b3f92f9)
### Screen shots:
![issue 4](https://user-images.githubusercontent.com/22956130/40482074-1c28a1e6-5f71-11e8-9ee8-700b75068277.png)

### Inspect Screenshots:
![screenshot 58](https://user-images.githubusercontent.com/22956130/40482229-b2cbf4f4-5f71-11e8-8d8c-0f72d80969f5.png)
![screenshot 59](https://user-images.githubusercontent.com/22956130/40482230-b2fe0c50-5f71-11e8-9624-b97ae506bc7a.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
